pkg/controller/constants.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+const (
+	Backup                = "backup"
+	BackupDeletion        = "backup-deletion"
+	BackupStorageLocation = "backup-storage-location"
+	BackupSync            = "backup-sync"
+	DownloadRequest       = "download-request"
+	GarbageCollection     = "garbage-collection"
+	PodVolumeBackup       = "pod-volume-backup"
+	PodVolumeRestore      = "pod-volume-restore"
+	ResticRepo            = "restic-repository"
```

Comment:
Changing this from `restic-repo` to `restic-repository` will change the value that can be passed to the `server --disable-controllers` flag which could be a breaking change for some users. Is the name significant when creating a `genericController`? Could we use `restic-repo` for the name?

Reply:
Good point. There's no big need to spell out the name, I'll change it back.